src/TableColumn/ActionButtons.php

```diff
@@ -100,7 +100,7 @@ public function addButton($button, $action = null, $confirm = false, $isDisabled
      * load contents through $callback. Will pass a virtual page.
      *
      * @param \atk4\ui\View|string $button
-     * @param string               $title
+     * @param string|array         $title - model title or model seed array
```

Comment:
There should be no `-`.

"model", what class is seeded by default?

Reply:
Corrected